### PR TITLE
starting to handle feature types appropriately (AGR 732)

### PR DESCRIPTION
**Pull request by nathandunn**
Fixes: 

https://agr-jira.atlassian.net/browse/AGR-732

### CODE REVIEW

**Comment by nathandunn** [comment]
FYI, @pjhale

File: src/containers/genePage/genomeFeatureViewer.js [hunk 1]
```diff
@@ -79,45 +82,32 @@ class GenomeFeatureViewer extends Component {
...
-              loadState: 'loaded'
-              , loadedData: data
-            });
-            return data;
-          });
-        })
-        .catch(() => {
-          // console.log(error);
+    this.setState({loadState: 'loading'});
+    this.transcriptTypes = getTranscriptTypes();
```
Comment (sibyl229) [inline src/containers/genePage/genomeFeatureViewer.js]: A minor thing here. If you need to work with constants in a React component, you don't have to make it an attribute of the component and define it in lifecycle. (Things that are neither prop or state is used very sparingly in React.) Simply define it before your class definition should work.

**Comment by nathandunn** [comment]
Thanks @sibyl229

**Comment by sibyl229** [comment]
Thanks @nathandunn !

**Pull request merged**